src/server/transaction.cc

```diff
@@ -1452,12 +1452,9 @@ OpResult<KeyIndex> DetermineKeys(const CommandId* cid, CmdArgList args) {
     if (!absl::SimpleAtoi(num, &num_custom_keys) || num_custom_keys < 0)
       return OpStatus::INVALID_INT;
 
-    if (name == "ZDIFF" && num_custom_keys == 0) {
```

Comment:
Did you intentionally remove ZDIFF? It seems to be unsupported as well

Reply:
thanks, fixed.